frontend/src/components/dashboard/myJobCents.jsx

```diff
@@ -106,7 +120,7 @@ export default class MyJobCents extends React.Component {
                     Send
                 </a>
                 <a
-                    title="Redeem"
+                    title="Redeem a challenge"
```

Comment:
also too verbose? how does it look on the tile?

Reply:
I thought it was more explanatory.  If they hover their mouse there, they probably want details on what the button will do.